### DISCUSSION TITLE
Enable graphical Mapping for transforming data

### DISCUSSION THREAD

**Discussion by samar-elsayed** [apache/camel-karavan#1418] [Ideas]
It would be really handy and productive for us ( at least in my company ) to have a graphical Mapping feature like the ones available in [Mule Soft Anypoint Studio ](https://docs.mulesoft.com/mule-runtime/latest/transform-graphically-construct-mapping-studio-task) and in [IBM App Connect Enterprise](https://www.ibm.com/docs/en/imdm/14.0?topic=elements-specifying-transform)

So please consider this idea for Karavan 
@mgubaidullin

**Comment by mgubaidullin** [comment]
https://github.com/apache/camel-karavan/discussions/693
